eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -37,13 +37,11 @@
           }
       },
       "TestMode": {
-          "netcoreapp21_Coverage_Record": {
+          "netcoreapp31_Coverage_Record": {
               "SupportsRecording": true,
               "CollectCoverage": true,
-              "TestTargetFramework": "netcoreapp3.1"
-          },
-          "net5.0": {
-              "TestTargetFramework": "net5.0"
+              "TestTargetFramework": "netcoreapp3.1",
+              "AdditionalTestArguments": "/p:UseProjectReferenceToAzureClients=false"
```

Comment:
Isn't this the default value? Do we need to explicitly pass it in this extra configuration?

Reply:
Didn't work without it the `$(AdditionalTestArguments)` literal value was passed to dotnet test.